File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/EmailSample/src/com/hp/workpath/sample/emailsample/model/EmailAddress.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.emailsample.model;

public class EmailAddress {
    private String address;
    private String name;

    public EmailAddress(String mailAddress, String mailName) {
        this.address = mailAddress;
        this.name = mailName;
    }

    public String getAddress() {
        return address;
    }

    public String getName() {
        return name;
    }
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/EmailSample/src/com/hp/workpath/sample/emailsample/model/EmailAddress.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.emailsample.model

class EmailAddress() {
    var address: String = ""
    var name: String? = null

    constructor(address: String, name: String?) : this() {
        this.address = address
        this.name = name
    }
}